src/tipowof10.c

```diff
@@ -41,6 +41,8 @@ const __int128 tipowof10[] = { 1ll, /* 10**0 */
   10000000000000000ll, /* 10**16 */
   100000000000000000ll, /* 10**17 */
   1000000000000000000ll, /* 10**18 */
+#ifdef __clang__
+#else
```

Comment:
`#ifndef __clang__` instead?
also, why? add a comment.

Reply:
#ifndef __clang__ sure

// Clang does support constant folding for __int128
// Simply not posible to defind a const larger than __UINT64_MAX__